### PR TITLE
W-13960337-persistence-gateway-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/persistence-gateway.adoc [hunk 1]
```diff
@@ -90,6 +90,18 @@ The output of this command should be similar to the following:
 192.168.2.101 - - [09/Apr/2021:16:35:55 +0000] "GET /api/v1/status/ready HTTP/1.1" 200 2 "" "kube-probe/1.18+"
 ----
 
+== Configure Authorize Namespace
+
+Edit the RoleBiding that you create during the configuration of xref:install-helm.adoc#optional-configure-authorized-namespaces[authorized namespaces] by adding a `subjects` object:
```
Comment (jiahongli18) [inline modules/ROOT/pages/persistence-gateway.adoc]:
```suggestion
Edit the RoleBinding that you create during the configuration of xref:install-helm.adoc#optional-configure-authorized-namespaces[authorized namespaces] by adding a `subjects` object:
```

Comment (jiahongli18) [reply modules/ROOT/pages/persistence-gateway.adoc]: They don't need to add the subjects field, it's already there. They just need to add a new subject.

File: modules/ROOT/pages/persistence-gateway.adoc [hunk 2]
```diff
@@ -90,6 +90,18 @@ The output of this command should be similar to the following:
 192.168.2.101 - - [09/Apr/2021:16:35:55 +0000] "GET /api/v1/status/ready HTTP/1.1" 200 2 "" "kube-probe/1.18+"
 ----
 
+== Configure Authorize Namespace
```
Comment (jiahongli18) [inline modules/ROOT/pages/persistence-gateway.adoc]: This section makes it seem like they need to configure authorized namespaces. This is not required. Only if they are already using authorized namespaces and they want to use persistence gateway, they need this step.

File: modules/ROOT/pages/persistence-gateway.adoc [hunk 3]
```diff
@@ -90,6 +90,18 @@ The output of this command should be similar to the following:
 192.168.2.101 - - [09/Apr/2021:16:35:55 +0000] "GET /api/v1/status/ready HTTP/1.1" 200 2 "" "kube-probe/1.18+"
 ----
 
+== Persistence Gateway and Authorized Namespaces
```
Comment (jiahongli18) [inline modules/ROOT/pages/persistence-gateway.adoc]:
```suggestion
== Persistence Gateway with Authorized Namespaces
```

**Review by jiahongli18 — APPROVED** [review]

**Review by tasmoore — APPROVED** [review]

**Pull request merged**